lib/scoped-models/cart.dart

```diff
@@ -199,7 +197,10 @@ mixin CartModel on Model {
     Map<String, String> headers = await getHeaders();
 
     Map<String, dynamic> paymentPayload = {
-      'payment': {'payment_method_id': 3, 'amount': order.itemTotal}
+      'payment': {
+        'payment_method_id': paymentMethodId,
```

Comment:
@sakina1403  is the payment method ID coming from back-end ?

Reply:
@gopalshimpi Not yet. I have the API for it, will integrate it tomorrow.

Reply:
Keep It in Mind Before deploying to Production.